package Module1.Task2.Exceptions;


public class ExecutionAlreadyCalledException extends RuntimeException{
}
